mypy.ini

```diff
@@ -87,10 +87,13 @@ ignore_errors = True
 [mypy-torch.quantization.fx.*]
 ignore_errors = True
 
-[mypy-torch.distributions.*]
+[mypy-torch._tensor_str]
 ignore_errors = True
 
-[mypy-torch._tensor_str]
+[mypy-torch.jit.quantized]
+ignore_errors = True
+
+[mypy-torch.nn.functional]
```

Comment:
are these actually intended to be added? or is this a merge master issue?

Reply:
They are added because after annotating the distributions.* dir, a lot of typing errors appear under these two modules. Maybe it's because master update. I am looking into this.